src/SchemaDesign/SchemaDesignerQuestion.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable dot-notation */
+import { SchemaQuestion } from '@/types/interfaces'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Stack from '@mui/material/Stack'
+import TextField from '@mui/material/TextField'
+import React from 'react'
+
+export default function SchemaDesignerQuestion({
+  onSubmit,
+  question,
+  closeQuestionPicker,
+}: {
+  onSubmit: (data: SchemaQuestion) => void
+  question: any // type this
+  closeQuestionPicker: () => void
+}) {
+  const [formInput, setFormInput] = React.useState<SchemaQuestion>({
+    title: '',
+    description: '',
+    type: question.type,
+    reference: '',
+    format: question.format || '',
+  })
+
+  const handleChange = (event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
+    const { value, name }: { value: string | number; name: string } = event.target
```

Comment:
By doing this we are overwriting the types with incorrect types. The default types are correct, both `value` and `name` will be of type `string` as TextFields will always emit change events with the `value` being a `string`, even if you provide a `number` as the `value` prop.

I'd suggest removing this line and just accessing the value and name via `event.target.value` and `event.target.name`.